Blockzilla/OverlayView.swift

```diff
@@ -218,34 +218,39 @@ class OverlayView: UIView {
         let searchSuggestionsPromptHidden = UserDefaults.standard.bool(forKey: SearchSuggestionsPromptView.respondedToSearchSuggestionsPrompt) || searchQuery.isEmpty
         var copyButtonHidden = true
 
-        UIPasteboard.general.urlAsync() { handoffUrl in
```

Comment:
@garvankeeley Any ideas on why we need to call handoffUrl with every key stroke change? 

Here I am proposing that we don't do that. For now I have simply commented out the code.

Reply:
no i am not familiar with this tbh